internal/pkg/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -352,7 +352,7 @@ func (u *ALMExamplesUpdate) Apply(csv *olmapiv1alpha1.ClusterServiceVersion) err
 		}
 	}
 	sb.WriteString(`]`)
-
-	csv.GetAnnotations()["alm-examples"] = sb.String()
+	prettyJSON, _ := json.MarshalIndent(sb, "", "\t")
```

Comment:
`sb` is already encoded JSON, so marshaling it will just return another long JSON string containing the escaped contents of `sb`. So I don't think this will do what you want. I think we need to just indent the JSON string, not marshal it. Take a look at [`json.Indent`](https://golang.org/pkg/encoding/json/#Indent).

Reply:
👍